### ISSUE TITLE
Missing missiles information

### ISSUE THREAD

**Issue by Araamarmorth** [spooky/unitdb#38]
There is currently no way to access missile's cost in mass, energy or build time: this could be accesses from TML, UEF and Seraphim com, nuclear launcher, Yolona Oss and nuke launching boats.

**Comment by spooky** [comment]
Indeed, but these numbers are not in the blueprint file, and I didn't find the place where this is available.

**Comment by Araamarmorth** [comment]
This can be found in the Economy section of the projectiles' blueprint.

For instance in projectiles -created from projectiles.nx2 and projectiles.scd-/AIFMissileTactical01/AIFMissileTactical01_proj.bp you can find:

    Economy = {
        BuildCostEnergy = 3600,
        BuildCostMass = 180,
        BuildTime = 2400,
    },

If you are interested I could a script to extract this on a per/weapon basis. (This could also be useful if one day we want to know which missiles follow their target or what are their acceleration).

**Comment by spooky** [comment]
Makes sense, this should probably be a part in the current tools. Queuing up in the pipeline.

**Comment by HopeAndWonder** [comment]
Hey spooky i am really new to java script, currently i am programming an android app for the UnitDataBase(i am learning how to programm android apps in general). Could you tell me how to extract the stats from the game so that i get a .json file? Sorry for contacting you in this way. If u dont have time for this, sorry for bothering you.

**Comment by spooky** [comment]
HopeAndWonder: In general, you need to extract 3 files (all zip files with different extension): units.scd (supcom:fa install dir), faforever.nxt, units.nx2 (both Faforever\gamedata dir). Then search for .bp files. These are lua files with the unit config.
A simpler way would be just to take the index.json (minified version, removed unused properties) or index.fat.json (full unit extract) from this repo
